jaeger-dropwizard/README.md

```diff
@@ -0,0 +1,91 @@
+# Dropwizard
+
+## Maven Dependency
+If you are using dropwizard then you can add the following:
+```xml
+<dependency>
+    <groupId>com.uber.jaeger</groupId>
+    <artifactId>jaeger-dropwizard</artifactId>
+    <version>$jaegerVersion</version>
+</dependency>
+```
+
+## Instrumentation
+This project includes instrumentation compatible with the JAX-RS standard. Implementations such as 
+Jersey and Dropwizard allow you to specify client and server filters to do some computational work 
+before every request or response. These filters are combined into a 
+JAX-RS [Feature](https://jersey.java.net/apidocs/2.9/jersey/javax/ws/rs/core/Feature.html).
+
+You will need to add a `JaegerFeature` to both the server and all JAX-RS clients that you use.
+If you use other clients, like an Apache Http Client, make sure to instrument them as well.
```

Comment:
here we should mention that it depends on jaxrs module and explain what this module adds on top of that